kong/plugins/oauth2/schema.lua

```diff
@@ -31,6 +31,7 @@ return {
     accept_http_if_already_terminated = { required = false, type = "boolean", default = false },
     anonymous = {type = "string", default = "", func = check_user},
     global_credentials = {type = "boolean", default = false},
+    auth_header_name = {required = false, type = "string", default = "authorization"},
```

Comment:
this will require a migration. Existing records in the datastore will not have this field, nor the default. (The default is not applied when loading from the datastore, but only when adding the plugin config)

This also means that this PR cannot go into `master` but has to go into `next` (migrations go in the next major release, which is the `next` branch)

As an example, look at the migrations in this PR: https://github.com/Kong/kong/pull/2883
(you need to rebase on `next` to get those files)